src/palm-react/hooks/app/useAlphaTest.ts

```diff
@@ -0,0 +1,98 @@
+import { UTIL } from 'palm-core/libs'
+import { useEffect, useState } from 'react'
+
+type AlphaConfig = {
+  waitlist: boolean
+}
+
+/* Hook used only in alpha test
+ * 1. fetch config.json from github assets
+ * 2. fetch waitlist.json from github assets
+ * 3. return config and waitlist
+ */
+const useAlphaTest = (): {
```

Comment:
useAlphaTest? doesn't get the naming. Maybe just useWaitlist?

Reply:
renamed